src/routers/handlers/stop-screen/stopScreenHandler.ts

```diff
@@ -73,6 +74,17 @@ const setContent = async (req: Request, stopType: STOP_TYPE, baseViewData: BaseV
                 extraData: [env.GET_RP01_LINK, env.GET_PSC01_LINK, env.POST_TO_CH_LINK, PrefixedUrls.START]
             };
         }
+        case STOP_TYPE.SUPER_SECURE: {
+            return {
+                ...baseViewData,
+                ...getLocaleInfo(locales, lang),
+                templateName: stopType,
+                currentUrl: resolveUrlTemplate(stopScreenPrefixedUrl, stopType),
```

Comment:
the currentUrl in this Super Secure block has lost the 'addSearchParams' function
`addSearchParams(getUrlWithStopType(stopScreenPrefixedUrl, stopType), { companyNumber, lang })`